x-pack/plugins/siem/public/components/page/hosts/host_summary/__snapshots__/index.test.tsx.snap

```diff
@@ -1,38 +1,887 @@
+            draggable="false"
+            style="z-index: 9000;"
+            tabindex="0"
+          >
+            <div
+              class="euiText euiText--small"
+              data-test-subj="draggable-content"
+            >
+              9 (stretch)
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <dt
+      class="euiDescriptionList__title"
+    >
+      Architecture
+    </dt>
+    <div>
+      <div
+        data-test-subj="draggableWrapperDiv"
+      >
+        <div
+          data-react-beautiful-dnd-droppable="6"
+        >
+          <div
+            aria-roledescription="Draggable item. Press space bar to lift"
+            class="sc-kEYyzF eIecMG"
+            data-react-beautiful-dnd-drag-handle="6"
+            data-react-beautiful-dnd-draggable="6"
+            data-test-subj="providerContainer"
+            draggable="false"
+            style="z-index: 9000;"
+            tabindex="0"
+          >
+            <div
+              class="euiText euiText--small"
+              data-test-subj="draggable-content"
+            >
+              x86_64
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </dl>
+</div>
+`;
+
 exports[`Host Summary Component rendering it renders the default Host Summary 1`] = `
 <Component>
   <pure(Component)
     data={
-      Array [
-        Object {
-          "cursor": Object {
-            "value": "aa7ca589f1b8220002f2fc61c64cfbf1",
-          },
-          "node": Object {
-            "_id": "yneHlmgBjVl2VqDlAjPR",
-            "host": Object {
-              "architecture": "x86_64",
-              "id": "aa7ca589f1b8220002f2fc61c64cfbf1",
-              "ip": Array [
-                "10.142.0.7",
-                "fe80::4001:aff:fe8e:7",
-              ],
-              "mac": Array [
-                "42:01:0a:8e:00:07",
-              ],
-              "name": "siem-kibana",
-              "os": Object {
-                "family": "debian",
-                "name": "Debian GNU/Linux",
-                "platform": "debian",
-                "version": "9 (stretch)",
-              },
-              "type": "projects/189716325846/machineTypes/n1-standard-1",
-            },
+      Object {
+        "_id": "yneHlmgBjVl2VqDlAjPR",
+        "host": Object {
+          "architecture": "x86_64",
+          "id": "aa7ca589f1b8220002f2fc61c64cfbf1",
+          "ip": Array [
+            "10.142.0.7",
+            "fe80::4001:aff:fe8e:7",
+          ],
+          "mac": Array [
+            "42:01:0a:8e:00:07",
+          ],
+          "name": "siem-kibana",
+          "os": Object {
+            "family": "debian",
+            "name": "Debian GNU/Linux",
+            "platform": "debian",
+            "version": "9 (stretch)",
           },
+          "type": "projects/189716325846/machineTypes/n1-standard-1",
         },
-      ]
+      }
     }
     endDate={618472800000}
     loading={false}
```

Comment:
This snapshot is definitely on the larger side at 887 lines. Nothing to change now as this component is getting refactored  as part of https://github.com/elastic/ingest-dev/issues/364 -- but that would be a fine time to revisit which of these tests are best suited to using snapshots.

Reply:
I think that's because we are having multiple snapshots in one file